src/vellum/workflows/events/types.py

```diff
@@ -46,9 +55,17 @@ def encode(obj: type) -> "CodeResourceDefinition":
         return CodeResourceDefinition(**serialize_type_encoder(obj))
 
 
+class VellumCodeResource(CodeResourceDefinition):
+    id: UUID
```

Comment:
[nit] ah I think _here_, we're safe to add directly to `CodeResourceDefinition`. It's on the vellum side that we'll want to split out into a separate class

Reply:
oh bet! that'll be a fast change